plugins/ocm/README.md

```diff
@@ -1,104 +1,114 @@
-              skipTLSVerify: # Skip TLS certificate verification, defaults to false
-              caData: # Base64-encoded CA bundle in PEM format
-      ```
-
-   2. The hub configuration can be tied to the kubernetes plugin configuration by providing the name of the hub. This is useful when you already use a kubernetes plugin in your backstage instance. Please add following to your `app-config.yaml` file:
-
-      ```yaml
-      # app-config.yaml
-      kubernetes:
-        serviceLocatorMethod:
-          type: "multiTenant"
-        clusterLocatorMethods:
-          - type: "config"
-            clusters:
-              - name: <cluster-name>
-              ...
-
-      catalog:
-        providers:
-          ocm:
-            env: # Key is reflected as provider ID. Defines and claims plugin instance ownership of entities
-              kubernetesPluginRef: <cluster-name> # Match the cluster name in kubernetes plugin config
-      ```
-
-      Please consult the documentation to [Backstage Kubernetes plugin](https://backstage.io/docs/features/kubernetes/configuration#configuring-kubernetes-clusters) for details on its configuration. Hub cluster must be connected via Service Account.
-
-3. Create new plugin instance in `packages/backend/src/plugins/ocm.ts`:
+2. Configure the OCM backend plugin using one of the following configurations:
+
+   - The OCM configuration provides the information about your hub. To use the OCM configuration, add the following code to your `app-config.yaml` file:
+
+     ```yaml
+     # app-config.yaml
+     catalog:
+       providers:
+         ocm:
+           env: # Key is reflected as provider ID. Defines and claims plugin instance ownership of entities
+             name: # Name that the hub cluster will assume in Backstage Catalog (in OCM this is always local-cluster which can be confusing)
+             url: # Url of the hub cluster API endpoint
+             serviceAccountToken: # Token used for querying data from the hub
+             skipTLSVerify: # Skip TLS certificate verification, defaults to false
+             caData: # Base64-encoded CA bundle in PEM format
+     ```
+
+   - You can bind the hub configuration to the Kubernetes configuration by providing the name of the hub as follows:
+
+     ```yaml
+     # app-config.yaml
+     kubernetes:
+       serviceLocatorMethod:
+         type: "multiTenant"
+       clusterLocatorMethods:
+         - type: "config"
+           clusters:
+             - name: <cluster-name>
+             ...
+
+     catalog:
+       providers:
+         ocm:
+           env: # Key is reflected as provider ID. Defines and claims plugin instance ownership of entities
+             kubernetesPluginRef: <cluster-name> # Match the cluster name in kubernetes plugin config
+     ```
+
+     This is useful when you already use a Kubernetes plugin in your Backstage instance. Also, the hub cluster must be connected using `ServiceAccount`.
```

Comment:
Same comment re capitalization of hub vs Hub. Same comment as previous re service account vs `ServiceAccount`.

Reply:
Referring to `ServiceAccount` object, which is similar to our other docs.

Reply:
in that case I'd specifically say "a `ServiceAccount` object" per the guidance in https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#api-object-formatting

Reply:
Sure, makes sense